src/services/transpile.ts

```diff
@@ -109,10 +110,14 @@ interface Symbol {
     readonly [Symbol.toStringTag]: string;
 }`;
 const barebonesLibName = "lib.d.ts";
-const barebonesLibSourceFile = createSourceFile(barebonesLibName, barebonesLibContent, { languageVersion: ScriptTarget.Latest });

```

Comment:
We should totally fix this on main regardless of this change; didn't realize this was unconditionally created.

Reply:
https://github.com/microsoft/TypeScript/pull/58931

Reply:
Stumbled into it  accidentally while looking at the megamorphism in tests, and could not for the life of me understand why there were still two node shapes, turns out this was created before the node allocator was swapped for the service one.

Reply:
Oh dang, that's actually an even worse regression.